src/main/java/com/todolist/enterprise/service/TodoServiceStub.java

```diff
@@ -7,16 +7,13 @@
 public class TodoServiceStub implements ITodoService {
     @Override
     public Task getTaskById(int taskId) {
-        if (taskId == 1) {
-            Task task = new Task();
-            task.setName("Task 1");
-            task.setDueDate(new Date());
-            task.setCompleted(false);
-            task.setListId(1);
 
-            return task;
-        }
-        
-        return null;
```

Comment:
This could return a null pointer exception if not handled properly.
[Short summary of Java Coding Pactices](https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3)